com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestSearchUtils.java

```diff
@@ -25,13 +34,76 @@
 import org.eclipse.jdt.ls.core.internal.JDTUtils;
 import org.eclipse.lsp4j.Range;
 
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 @SuppressWarnings("restriction")
 public class TestSearchUtils {
+    private static final Map<TestLevel, TestItemSearcher[]> searcherMap = initializeSearcherMap();
+
+    private static Map<TestLevel, TestItemSearcher[]> initializeSearcherMap() {
+        final Map<TestLevel, TestItemSearcher[]> map = new HashMap<TestLevel, TestItemSearcher[]>();
+        map.put(TestLevel.FOLDER, new TestItemSearcher[] {new PackageSearcher()});
+        map.put(TestLevel.PACKAGE, new TestItemSearcher[] {new ClassSearcher() });
```

Comment:
An extra white space here. Do you have related checkstyle rules for it?

Reply:
Not yet, but will consider adding more check rules in the future.